app/models/ext_management_system/supports_attribute.rb

```diff
@@ -27,12 +27,14 @@ module ExtManagementSystem::SupportsAttribute
     def supports_attribute(colname = nil, feature: :create, child_model: nil)
       feature = feature.to_sym
 
+      colname ||= "supports_#{feature}"
```

Comment:
NOTE I wasn't sure if child_model != nil implied colname had to be defined

Reply:
please move down next to where it was defined below around 37